workers/pull_request_worker/pull_request_worker/worker.py

```diff
@@ -790,6 +798,49 @@ def query_pr_comments(self, owner, repo, gh_pr_no, pr_id):
 
         logging.info(f'Finished adding PR Message data for PR with id {pr_id}')
 
+    def query_pr_repo(self, pr_repo, pr_repo_type, pr_meta_id):
+        logging.info(f'Querying PR {pr_repo_type} repo')
+
+        pseudo_key_gh = 'id'
+        psuedo_key_augur = 'pr_src_repo_id'
+        table = 'pull_request_repo'
+        pr_repo_table_values = self.get_table_values({psuedo_key_augur: pseudo_key_gh}, [table])
+
+        new_pr_repo = self.check_duplicates([pr_repo], pr_repo_table_values, pseudo_key_gh)
+
+        if new_pr_repo:
+            if new_pr_repo[0]['owner'] and 'login' in new_pr_repo[0]['owner']:
+                cntrb_id = self.find_id_from_login(new_pr_repo[0]['owner']['login'])
+            else:
+                cntrb_id = 1
+
+            pr_repo = {
+                'pr_repo_meta_id': pr_meta_id,
+                'pr_repo_head_or_base': pr_repo_type,
+                'pr_src_repo_id': new_pr_repo[0]['id'],
+                # 'pr_src_node_id': new_pr_repo[0]['node_id'],
```

Comment:
@parthsharma2 : Are we going to add the src fork data in a different pull request?

Reply:
If by src fork data you mean, the HEAD & BASE repos, then no. The `query_pr_repo()` method inserts the HEAD & BASE repos data into the `pull_request_repo` table.

Reply:
@parthsharma2 : is that new? I don't see it populated (the head and base) in the current version.  Is this in the PR or is it supposed to already be there?

Reply:
The current version adds the head and base metadata to the `pull_request_meta` table. This PR adds the head and base repo data to the `pull_request_repo` table.